client/src/pages/characterSearch/CharacterSearch.js

```diff
@@ -1,31 +1,32 @@
-import React, { useRef, useState } from 'react'
-import axios from 'axios';
+import React, { useEffect, useRef, useState } from 'react'
 
 import Styles from './characterSearch.module.css'
-import { Character } from '../../componenets/Character';
+import Character from '../../componenets/Character';
```

Comment:
for avoiding all the ../../ U can adding config for the path.
for page, component, assest, etc.